internal/api/handlers.go

```diff
@@ -70,7 +70,7 @@ func (h *Handler) ApiTaskCreate(w http.ResponseWriter, r *http.Request) {
 func (h *Handler) ApiTasksGetAll(w http.ResponseWriter, r *http.Request) {
 	tasks, err := h.service.GetAllTasks()
 	if err != nil {
-		response := map[string]string{"error": fmt.Sprintf("%s", err.Error())}
+		response := map[string]string{"error": fmt.Sprintf("%v", err)}

```

Comment:
Я может в прошлый раз упустил, но тут даже можно так
```suggestion
		response := map[string]string{"error": err.Error()}
```